composer.json

```diff
@@ -68,12 +73,32 @@
 	"autoload-dev": {
 		"psr-4": { "Imagify\\Tests\\": "Tests/"	}
 	},
+	"extra": {
+		"mozart": {
+			"dep_namespace": "ImagifyPlugin\\Dependencies\\",
+			"dep_directory": "/src/Dependencies/",
+			"classmap_directory": "/classes/Dependencies/",
+			"classmap_prefix": "WPMIP_",
+			"packages": [
+				"deliciousbrains/wp-background-processing"
+			],
+			"delete_vendor_directories": true
```

Comment:
The default  value is already `true` for this parameters, so we don't need to add it